Formula/p/pngquant.rb

```diff
@@ -21,8 +21,15 @@ class Pngquant < Formula
   depends_on "libpng"
   depends_on "little-cms2"
 
+  resource "manpage" do
+    url "https://raw.githubusercontent.com/kornelski/pngquant/4140cc5bc9f48f4df044754bef6cf784e1f645f8/pngquant.1"
```

Comment:
Can we use the version from a matching Git tag?

Reply:
yeah, I can.